openscm/scmdataframebase.py

```diff
@@ -532,6 +532,28 @@ def set_meta(self, meta, name=None, index=None):
         )
         self._sort_meta_cols()
 
+    def line_plot(self, x='year', y='value', **kwargs):
```

Comment:
```suggestion
    def line_plot(self, x='time', y='value', **kwargs):
```

We don't have 'year' in ScmDataFrame